checker/localizations_src/en/messages.yaml

```diff
@@ -9,9 +9,9 @@ request-property-pattern-added: "added the pattern '%s' for the request property
 request-property-pattern-changed: "changed the pattern for the request property %s from '%s' to '%s'"
 api-deprecated-sunset-parse: api sunset date can't be parsed for deprecated API
 api-sunset-date-too-small: "api sunset date %s is too small, must be at least %d days from now"
-api-path-added: api path added
-api-path-deprecated: api path deprecated
-api-path-reactivated: api path reactivated
+endpoint-added: endpoint added
```

Comment:
I think "endpoint" is more intuitive than "api path".
Let's update the older messages in a separate PR because it could break "ignore" configuration files